nemo_curator/classifiers/domain.py

```diff
@@ -36,10 +37,15 @@ class DomainModelConfig:
 
 
 class DomainModel(HFModel):
-    def __init__(self, config: DomainModelConfig, autocast: bool = False):
+    def __init__(
+        self, config: DomainModelConfig, autocast: bool = False, max_mem_gb=None
+    ):
         self.config = config
         self.autocast = autocast
-        super().__init__(self.config.model)
+        if max_mem_gb is None:
+            max_mem_gb = _get_suggest_memory_for_classifier()
```

Comment:
I like this memory method. Can you add a docstring to the `DomainClassifer` and `QualityClassifier` and explain what the max memory does and how it will be calculated if it is set to `None`?

Reply:
Cool, will do . Let me add it to the docstring.

Reply:
Resolved by adding doc-string to https://github.com/NVIDIA/NeMo-Curator/pull/168/commits/e889f51a9f8f1fbe77f9ce256cb378c0f3083e32